src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -314,4 +314,40 @@ describe('Test suite for dictionary concepts components', () => {
     wrapper.find('.cancelButton').simulate('click');
     expect(createConceptInstance.state.show).toEqual(false);
   });
+
+  it('should call updateSourceEventListener function when setting the source to a new mapping that is not `CIEL`', () => {
```

Comment:
I do not seem to understand testing that updateSourceEventListener is called yet when you are explicitly calling it yourself in the test.

Reply:
I have adjusted this description accordingly